Controller/ThemeController.php

```diff
@@ -40,18 +41,32 @@ class ThemeController
      */
     protected $cookieOptions;
 
+    /**
+     * @var RouterInterface
+     */
+    protected $router;
+
+    /**
+     * @var string
+     */
+    protected $defaultRoute;
+
     /**
      * Theme controller construct.
      *
      * @param ActiveTheme $activeTheme   active theme instance
      * @param array       $themes        Available themes
      * @param array|null  $cookieOptions The options of the cookie we look for the theme to set
+     * @param RouterInterface $router
+     * @param string $defaultRoute
      */
-    public function __construct(ActiveTheme $activeTheme, array $themes, array $cookieOptions = null)
+    public function __construct(ActiveTheme $activeTheme, array $themes, array $cookieOptions = null, RouterInterface $router, $defaultRoute = '/')
```

Comment:
default route probably should be the name of an actual route now, no?

so I think what we should probably do us to allow null for $defaultRoute and then when we do the redirect, check if its null and in that case fallback to the `/`

Reply:
Yeah, that's a good idea. I will change that. :)